Source/WebCore/dom/Document.cpp

```diff
@@ -6043,6 +6043,16 @@ void Document::updateEventRegions()
     }
 }
 
+EventTrackingRegions Document::getTouchEventRegions()
```

Comment:
`Document::getTouchEventRegions() const`.

We generally don't say "get". Just `touchEventRegions()` should be fine.

Reply:
Fixed.